CMakeLists.txt

```diff
@@ -78,16 +78,16 @@ option(VIAMCPPSDK_USE_DYNAMIC_PROTOS "Regenerate protos as part of building (net
 # Enforce known toolchains and toolchain minima unless asked not to.
 if (VIAMCPPSDK_ENFORCE_COMPILER_MINIMA)
   if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 10.2)
-      message(FATAL_ERROR "Insufficient GCC version: 10.2+ required")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.4)
```

Comment:
Downgrade to the clang/gcc compiler versions as found on focal and the XCode version associated with that clang version.